packages/riipen-ui/src/components/Tabs.test.jsx

```diff
@@ -0,0 +1,233 @@
+      const child1 = <Tab value="one" />;
+      const child2 = <Tab value="two" />;
+
+      const wrapper = mount(
+        <Tabs>
+          {child1}
+          {child2}
+        </Tabs>
+      );
+
+      const childrenNodes = wrapper.find("Tab");
+      expect(childrenNodes.at(0).props().value).toEqual("one");
+      expect(childrenNodes.at(1).props().value).toEqual("two");
+    });
+  });
+
+  describe("classes prop", () => {
+    it("applies all classes to the root node", () => {
+      const classes = ["classOne"];
+
+      const wrapper = mount(<Tabs classes={classes} />);
+
+      expect(
+        wrapper
+          .find("Tabs")
+          .childAt(0)
+          .hasClass(classes[0])
+      ).toEqual(true);
+    });
+
+    it("appends higher order values to classes prop with withClass decorator", () => {
+      const classes = ["riipen", "riipen-tabs"];
+
+      const wrapper = mount(<Tabs />);
+
+      expect(
+        wrapper
+          .find("Tabs")
+          .props()
+          .classes.sort()
+      ).toEqual(classes.sort());
+    });
+
+    it("gives an error when classes are provided as one string", () => {
+      const classes = "classOne classTwo";
+      const errors = jest.spyOn(console, "error").mockImplementation();
+
+      mount(<Tabs classes={classes} />);
+
+      expect(errors).toHaveBeenCalled();
+    });
+  });
+
+  describe("color prop", () => {
+    it("passes valid given color to children", () => {
+      const color = "primary";
+      const child = <Tab label="Item one" value="one" />;
+
+      const wrapper = mount(<Tabs color={color}>{child}</Tabs>);
+
+      expect(wrapper.find("Tab").props().color).toEqual(color);
+    });
+
+    it("gives an error with invalid color", () => {
+      const errors = jest.spyOn(console, "error").mockImplementation();
+
+      mount(<Tabs color="yellow" />);
+
+      expect(errors).toHaveBeenCalled();
+    });
+  });
+
+  describe("component prop", () => {
+    it("applies given component to the root node", () => {
+      const component = "h3";
+
+      const wrapper = mount(<Tabs component={component} />);
+
+      expect(wrapper.find("Tabs").childAt());
+    });
+
+    it("gives an error when given an invalid component", () => {
+      const errors = jest.spyOn(console, "error").mockImplementation();
+      const component = "abcd";
+
+      mount(<Tabs component={component} />);
+
+      expect(errors).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe("onChange prop", () => {
+    it("passes valid handler to the onClick prop of children nodes", () => {
+      const onChange = jest.fn();
+      const child = <Tab label="Item one" value="one" />;
+
+      const wrapper = mount(<Tabs onChange={onChange}>{child}</Tabs>);
+
+      wrapper.find("Tab").simulate("click");
+
+      expect(onChange).toHaveBeenCalledWith(expect.any(Object), "one");
+    });
+  });
+
+  describe("orientation prop", () => {
+    it("passes given valid orientation to children", () => {
```

Comment:
I think we should add a second test that checks for the `orientation` prop is also used in the `className` found on line 94.